packages/diagrams/frontend/sirius-components-diagrams/src/sprotty/views/ImageView.tsx

```diff
@@ -72,6 +73,34 @@ export class ImageView extends RectangularNodeView {
     }
     const selectedHandles = createResizeHandles(node);
 
+    const debugFontSize = 6;
+    const styleDebug = {
+      fontSize: debugFontSize + 'px',
+      fill: 'green',
+    };
+
+    const debugInfos = (
+      node: { root: Diagram; bounds: { x: number; y: number } },
+      rectangleSize: { width: number; height: number }
+    ) => {
+      const diagram = node.root;
+      if (diagram.debug) {
+        return (
+          <g>
+            <text x={0} y={0 + debugFontSize} style={styleDebug}>
+              x:{node.bounds.x.toFixed(2)} y:{node.bounds.y.toFixed(2)}
+            </text>
+            <text x={rectangleSize.width} y={rectangleSize.height - debugFontSize} style={styleDebug}>
+              Width: {rectangleSize.width.toFixed(2)}
+            </text>
+            <text x={0} y={rectangleSize.height - debugFontSize} style={styleDebug}>
+              Height: {rectangleSize.height.toFixed(2)}
+            </text>
+          </g>
+        );
+      } else return null;
+    };
+
```

Comment:
Do not create a function like that inline, it would be recreated every time the component is rendered